examples/code-splitting-native-import-context/README.md

```diff
@@ -0,0 +1,394 @@
+# example.js
+
+This example illustrates how to leverage the `import()` syntax to create ContextModules which are separated into separate chunks for each module in the `./templates` folder.
+
+``` javascript
+
+async function getTemplate(templateName) {
+	try {
+		let template = await import(`./templates/${templateName}`);
+		console.log(template);
+	} catch(err) {
+		console.error("template error");
+		return new Error(err);
+	}
+}
+
+getTemplate("a");
+getTemplate("b");
+getTemplate("c");
+```
+
+# templates/
+
+* a.js
+* b.js
+* c.js
+
+All templates are of this pattern:
+
+``` javascript
+module.exports = function() {
+	return "This text was generated by template X";
+}
```

Comment:
It's not yet 100% defined, but spec is leaning towards only mapping the import to `default`

https://github.com/airbnb/babel-plugin-dynamic-import-webpack/issues/12

At any rate, it would always be more compatible to be strict. If the spec is relaxed later, the rule might be relaxed without big issues, but making it stricter would be semver-major.